android/src/main/java/com/swmansion/rnscreens/ScreenStackFragment.kt

```diff
@@ -163,6 +175,19 @@ class ScreenStackFragment : ScreenFragment, ScreenStackFragmentWrapper {
         }
     }
 
+    private fun findLastFocusedChild(): View? {
+        // If current device is not television, we don't want to save last focused child.
+        if (!isTelevision()) return null
```

Comment:
```suggestion
```

Reply:
Changed in https://github.com/software-mansion/react-native-screens/pull/1894/commits/4466f8d6df6cac984534a13781bbc6f94f0491ec.